index.html

```diff
@@ -27,18 +27,37 @@ <h1 id="trybewarts-header-title">Trybewarts</h1>
       <!-- requisito 12 -->
       <div class="familias">
         <label id="label-family">Qual sua família?</label>
-        <input name="family" type="radio" value="Frontend"> Frontend
-        <input name="family" type="radio" value="Backend"> Backend
-        <input name="family" type="radio" value="FullStack"> FullStack
-        <select name="" id="house">
+        <label for="Frontend">Frontend</label>
+        <input name="family" type="radio" value="Frontend">
+        <label for="Backend">Backend</label>
+        <input name="family" type="radio" value="Backend"> 
+        <label for="FullStack">FullStack</label>
+        <input name="family" type="radio" value="FullStack"> 
```

Comment:
O `<label>` precisa ter o mesmo valor em `for=` que o `<input>` tem em `name=`
```suggestion
        <label for="frontend">Frontend</label>
        <input name="frontend" type="radio" value="Frontend">
        <label for="backend">Backend</label>
        <input name="backend" type="radio" value="Backend"> 
        <label for="fullstack">FullStack</label>
        <input name="fullstack" type="radio" value="FullStack"> 
```
Assim ao clicar com o mouse no label também seleciona a opção desejada